Day-I-Social-Card/social-card/src/components/HeaderComponents/Header.css

```diff
@@ -1 +1,38 @@
 /* All styles for your header Components live here */
+.header-container {
```

Comment:
Don't forget you can still use CSS specificity to target your element. Just declare the class first and work your way down.